### PR TITLE
Updated arrondissement names in France

### PR DESCRIPTION
**Pull request by stepps00**
Fixes https://github.com/whosonfirst-data/whosonfirst-data/issues/564. (Replaces: this [outdated branch](https://github.com/whosonfirst-data/whosonfirst-data/tree/stepps00/france-names)).

This PR includes updated `wof:name` fields for 71 arrondissement records in France. These records have either non-ASCII characters in the `wof:name` property or incorrect formatting. The original `wof:name` values were stored in the `fra_x_preferred` property, so all original names were retained in each record.

### CODE REVIEW

**Comment by nvkelso** [comment]
These are often neighbourhood labels that clutter up Mapzen base maps right now.

screen shot 2016-12-19 at 15 25 20

- Default to shortest possible name in each language
- Include longer name in each language as variant
- Strip off the locality from each name

For Paris:

* `420782683` 1er Arrondissement
* `420782685` 2ème Arrondissement
* `420782687` 3ème Arrondissement
* `420782689` 4ème Arrondissement
* `420782691` 5ème Arrondissement
* `420782693` 6ème Arrondissement
* `420782695` 7ème Arrondissement
* `420782699` 8ème Arrondissement
* `420782701` 9ème Arrondissement
* `420782703` 10ème Arrondissement
* `420782705` 11ème Arrondissement
* `420782707` 12ème Arrondissement
* `420782709` 13ème Arrondissement
* `420782711` 14ème Arrondissement
* `420782713` 15ème Arrondissement
* `420782717` 16ème Arrondissement
* `420782719` 17ème Arrondissement
* `420782721` 18ème Arrondissement
* `420782723` 19ème Arrondissement
* `420782725` 20ème Arrondissement

This would work out to:

* `420782683` 1er Arrondissement
  - eng preferred: `1st Arr.`
  - fra preferred: `1er Arr.`
  - eng variant: `1st Arrondissement`
  - fra variant: `1er Arrondissement`
* `420782685` 2ème Arrondissement
  - eng preferred: `2nd Arr.`
  - fra preferred: `2e Arr.`
  - eng variant: `2nd Arrondissement`
  - fra variant: `2ème Arrondissement`
* `420782687` 3ème Arrondissement
  - eng preferred: `3rd Arr.`
  - fra preferred: `3e Arr.`
  - eng variant: `3rd Arrondissement`
  - fra variant: `3ème Arrondissement`
* `420782689` 4ème Arrondissement
  - eng preferred: `4th Arr.`
  - fra preferred: `4e Arr.`
  - eng variant: `4th Arrondissement`
  - fra variant: `4ème Arrondissement`
* `420782691` 5ème Arrondissement
  - eng preferred: `5th Arr.`
  - fra preferred: `5e Arr.`
  - eng variant: `5th Arrondissement`
  - fra variant: `5ème Arrondissement`

... and so on.... (only the 1st is unique in French, and 1st, 2nd, 3rd in English)

(We might consider doing superscript for these in the future, but not now.)

**Comment by stepps00** [comment]
@nvkelso - The original `wof:name` should be retained as a variant name too, correct? 

For example, `420782683` would have the following values:

  - eng preferred: `1st Arr.`
  - fra preferred: `1er Arr.`
  - eng variant: `1st Arrondissement`
  - fra variant: `1er Arrondissement` **and** `Paris-1Er-Arrondissement`
  - "wof:name": `1st Arrondissement`

**Comment by nvkelso** [comment]
Yes

**Comment by stepps00** [comment]
Most recent [commit](c35d2ac) updated the English and French `name` fields for Arrondissements in France. Used the above rules to update `variant` and `preferred fields. 

Over to @nvkelso for a second glance.

File: data/102/067/591/102067591.geojson [hunk 1]
```diff
@@ -14,6 +14,19 @@
     "mps:latitude":48.854205,
     "mps:longitude":2.357364,
     "mz:hierarchy_label":1,
+    "name:eng_x_variant":[
+        "4th Arrondissement of Paris"
+    ],
+    "name:feng_x_preferred":[
```
Comment (nvkelso) [inline data/102/067/591/102067591.geojson]: [Ferengi](https://en.wikipedia.org/wiki/Ferengi)? 

Probably: `"name:eng_x_preferred":[`

Comment (nvkelso) [inline data/102/067/591/102067591.geojson]: (every record)

File: data/102/067/017/102067017.geojson [hunk 2]
```diff
@@ -51,8 +64,8 @@
         }
     ],
     "wof:id":102067017,
-    "wof:lastmodified":1466628153,
-    "wof:name":"Paris-1Er-Arrondissement",
+    "wof:lastmodified":1484009253,
+    "wof:name":"1st Arrondissement of Paris",
```
Comment (nvkelso) [inline data/102/067/017/102067017.geojson]: Drop ` of Paris` here, and & etc on other default `"wof:name"` properties values (it's already represented in the name variants for each language).

File: data/890/420/195/890420195.geojson [hunk 3]
```diff
@@ -14,6 +14,19 @@
...
+    ],
+    "name:feng_x_preferred":[
+        ""
+    ],
+    "name:fra_x_preferred":[
+        ""
+    ],
+    "name:fra_x_variant":[
+        "Arrondissement de Pointe-\u00e0-Pitre",
+        ""
```
Comment (nvkelso) [inline data/890/420/195/890420195.geojson]: Remove empty string variant.

File: data/890/420/195/890420195.geojson [hunk 4]
```diff
@@ -14,6 +14,19 @@
     "gn:pop":"211130",
     "iso:country":"GP",
     "mz:hierarchy_label":1,
+    "name:eng_x_variant":[
+        "Arrondissement de Pointe-a-Pitre"
+    ],
+    "name:feng_x_preferred":[
```
Comment (nvkelso) [inline data/890/420/195/890420195.geojson]: Remove entire empty preferred name variant block (since it's empty string).

Comment (nvkelso) [inline data/890/420/195/890420195.geojson]: Or set the preferred value to the current variant value, and ditch the variant instead.

File: data/890/420/195/890420195.geojson [hunk 5]
```diff
@@ -14,6 +14,19 @@
...
+        ""
+    ],
+    "name:fra_x_preferred":[
```
Comment (nvkelso) [inline data/890/420/195/890420195.geojson]: Ditto

File: data/404/428/863/404428863.geojson [hunk 6]
```diff
@@ -52,8 +65,8 @@
         }
     ],
     "wof:id":404428863,
-    "wof:lastmodified":1479932235,
-    "wof:name":"Lyon--9E--Arrondissement",
+    "wof:lastmodified":1484009253,
+    "wof:name":"9th Arrondissement of Lyon",
```
Comment (nvkelso) [inline data/404/428/863/404428863.geojson]: Another example to remove ` of Lyon`.

File: data/404/428/841/404428841.geojson [hunk 7]
```diff
@@ -52,8 +65,8 @@
         }
     ],
     "wof:id":404428841,
-    "wof:lastmodified":1479911391,
-    "wof:name":"Marseille-15E--Arrondissement",
+    "wof:lastmodified":1484009253,
+    "wof:name":"15th Arrondissement of Marseille",
```
Comment (nvkelso) [inline data/404/428/841/404428841.geojson]: Another example to remove ` of Marseille`

File: data/890/420/195/890420195.geojson (hunk 5 above)
Comment (nvkelso) [inline data/890/420/195/890420195.geojson]: This remains outstanding.

File: data/890/420/195/890420195.geojson (hunk 4 above)
Comment (nvkelso) [inline data/890/420/195/890420195.geojson]: This remains outstanding.

File: data/890/420/195/890420195.geojson [hunk 8]
```diff
@@ -14,6 +14,12 @@
     "gn:pop":"211130",
     "iso:country":"GP",
     "mz:hierarchy_label":1,
+    "name:eng_x_variant":[
```
Comment (nvkelso) [inline data/890/420/195/890420195.geojson]: Add a `"name:eng_x_prefered"` with `Arr. de Pointe-a-Pitre`

File: data/890/420/195/890420195.geojson [hunk 9]
```diff
@@ -14,6 +14,12 @@
...
+        "Arrondissement de Pointe-a-Pitre"
+    ],
+    "name:fra_x_variant":[
```
Comment (nvkelso) [inline data/890/420/195/890420195.geojson]: Add a `"name:fra_x_variant"` with `Arr. de Pointe-\u00e0-Pitre`

**Comment by nvkelso** [comment]
@stepps00 After the final two changes requested for `Arrondissement de Pointe-a-Pitre` you're cleared for takeoff.

**Comment by stepps00** [comment]
@nvkelso - should be clear for takeoff via https://github.com/whosonfirst-data/whosonfirst-data/commit/2d96e09f9e903968a97995ff16ee5dd1fecf9a33

**Comment by nvkelso** [comment]
For merging: Name changes only, no PIPing.

**Pull request merged**